### PR TITLE
The outline and the filled part of MicPulseOff were both filled. Fixing.

### PR DESCRIPTION
**Pull request by sfilipi**
## Current Behavior

MicPulseOff path for outline and filled is the same

Fixes #
Fixes the MicPulseOff outline path per what is in the Figma file. 

image

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/c0b27cce0d29122e4488ff431689b966076a728b'>c0b27cce0d29122e4488ff431689b966076a728b</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## Perf Analysis (`@fluentui/react-northstar`)
<details><summary>Perf tests with no regressions</summary>

Scenario | Current PR Ticks | Baseline Ticks | Ratio
:--- | ---:| ---:| ---:
CarouselMinimalPerf.default | [512](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/CarouselMinimalPerf.default.html) | [463](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/CarouselMinimalPerf.default_base.html) | 1.11:1
HeaderSlotsPerf.default | [828](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/HeaderSlotsPerf.default.html) | [751](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/HeaderSlotsPerf.default_base.html) | 1.1:1
AttachmentMinimalPerf.default | [162](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/AttachmentMinimalPerf.default.html) | [149](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/AttachmentMinimalPerf.default_base.html) | 1.09:1
GridMinimalPerf.default | [342](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/GridMinimalPerf.default.html) | [318](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/GridMinimalPerf.default_base.html) | 1.08:1
ButtonSlotsPerf.default | [601](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ButtonSlotsPerf.default.html) | [561](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ButtonSlotsPerf.default_base.html) | 1.07:1
ChatWithPopoverPerf.default | [394](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ChatWithPopoverPerf.default.html) | [368](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ChatWithPopoverPerf.default_base.html) | 1.07:1
FlexMinimalPerf.default | [297](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/FlexMinimalPerf.default.html) | [277](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/FlexMinimalPerf.default_base.html) | 1.07:1
FormMinimalPerf.default | [424](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/FormMinimalPerf.default.html) | [398](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/FormMinimalPerf.default_base.html) | 1.07:1
AvatarMinimalPerf.default | [199](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/AvatarMinimalPerf.default.html) | [188](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/AvatarMinimalPerf.default_base.html) | 1.06:1
MenuMinimalPerf.default | [884](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/MenuMinimalPerf.default.html) | [832](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/MenuMinimalPerf.default_base.html) | 1.06:1
StatusMinimalPerf.default | [708](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/StatusMinimalPerf.default.html) | [665](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/StatusMinimalPerf.default_base.html) | 1.06:1
TableManyItemsPerf.default | [2100](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/TableManyItemsPerf.default.html) | [1977](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/TableManyItemsPerf.default_base.html) | 1.06:1
TreeWith60ListItems.default | [184](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/TreeWith60ListItems.default.html) | [174](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/TreeWith60ListItems.default_base.html) | 1.06:1
DividerMinimalPerf.default | [388](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/DividerMinimalPerf.default.html) | [371](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/DividerMinimalPerf.default_base.html) | 1.05:1
DropdownManyItemsPerf.default | [742](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/DropdownManyItemsPerf.default.html) | [710](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/DropdownManyItemsPerf.default_base.html) | 1.05:1
DropdownMinimalPerf.default | [3277](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/DropdownMinimalPerf.default.html) | [3128](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/DropdownMinimalPerf.default_base.html) | 1.05:1
HeaderMinimalPerf.default | [384](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/HeaderMinimalPerf.default.html) | [366](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/HeaderMinimalPerf.default_base.html) | 1.05:1
InputMinimalPerf.default | [1400](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/InputMinimalPerf.default.html) | [1337](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/InputMinimalPerf.default_base.html) | 1.05:1
ItemLayoutMinimalPerf.default | [1238](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ItemLayoutMinimalPerf.default.html) | [1182](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ItemLayoutMinimalPerf.default_base.html) | 1.05:1
ButtonOverridesMissPerf.default | [1591](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ButtonOverridesMissPerf.default.html) | [1531](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ButtonOverridesMissPerf.default_base.html) | 1.04:1
LayoutMinimalPerf.default | [375](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/LayoutMinimalPerf.default.html) | [360](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/LayoutMinimalPerf.default_base.html) | 1.04:1
ListWith60ListItems.default | [691](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ListWith60ListItems.default.html) | [662](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ListWith60ListItems.default_base.html) | 1.04:1
LoaderMinimalPerf.default | [748](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/LoaderMinimalPerf.default.html) | [721](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/LoaderMinimalPerf.default_base.html) | 1.04:1
PortalMinimalPerf.default | [182](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/PortalMinimalPerf.default.html) | [175](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/PortalMinimalPerf.default_base.html) | 1.04:1
SegmentMinimalPerf.default | [340](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/SegmentMinimalPerf.default.html) | [327](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/SegmentMinimalPerf.default_base.html) | 1.04:1
VideoMinimalPerf.default | [676](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/VideoMinimalPerf.default.html) | [653](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/VideoMinimalPerf.default_base.html) | 1.04:1
SliderMinimalPerf.default | [1711](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/SliderMinimalPerf.default.html) | [1665](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/SliderMinimalPerf.default_base.html) | 1.03:1
IconMinimalPerf.default | [617](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/IconMinimalPerf.default.html) | [601](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/IconMinimalPerf.default_base.html) | 1.03:1
TreeMinimalPerf.default | [869](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/TreeMinimalPerf.default.html) | [841](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/TreeMinimalPerf.default_base.html) | 1.03:1
MenuButtonMinimalPerf.default | [1790](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/MenuButtonMinimalPerf.default.html) | [1756](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/MenuButtonMinimalPerf.default_base.html) | 1.02:1
ProviderMergeThemesPerf.default | [1292](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ProviderMergeThemesPerf.default.html) | [1265](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ProviderMergeThemesPerf.default_base.html) | 1.02:1
ToolbarMinimalPerf.default | [996](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ToolbarMinimalPerf.default.html) | [979](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ToolbarMinimalPerf.default_base.html) | 1.02:1
AnimationMinimalPerf.default | [555](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/AnimationMinimalPerf.default.html) | [551](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/AnimationMinimalPerf.default_base.html) | 1.01:1
ButtonMinimalPerf.default | [172](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ButtonMinimalPerf.default.html) | [170](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ButtonMinimalPerf.default_base.html) | 1.01:1
ChatMinimalPerf.default | [778](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ChatMinimalPerf.default.html) | [773](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ChatMinimalPerf.default_base.html) | 1.01:1
EmbedMinimalPerf.default | [4447](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/EmbedMinimalPerf.default.html) | [4419](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/EmbedMinimalPerf.default_base.html) | 1.01:1
ReactionMinimalPerf.default | [375](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ReactionMinimalPerf.default.html) | [372](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ReactionMinimalPerf.default_base.html) | 1.01:1
AttachmentSlotsPerf.default | [1104](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/AttachmentSlotsPerf.default.html) | [1104](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/AttachmentSlotsPerf.default_base.html) | 1:1
RosterPerf.default | [1107](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/RosterPerf.default.html) | [1106](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/RosterPerf.default_base.html) | 1:1
PopupMinimalPerf.default | [629](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/PopupMinimalPerf.default.html) | [631](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/PopupMinimalPerf.default_base.html) | 1:1
TooltipMinimalPerf.default | [1174](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/TooltipMinimalPerf.default.html) | [1173](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/TooltipMinimalPerf.default_base.html) | 1:1
DatepickerMinimalPerf.default | [6347](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/DatepickerMinimalPerf.default.html) | [6381](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/DatepickerMinimalPerf.default_base.html) | 0.99:1
ListMinimalPerf.default | [528](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ListMinimalPerf.default.html) | [534](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ListMinimalPerf.default_base.html) | 0.99:1
SplitButtonMinimalPerf.default | [4558](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/SplitButtonMinimalPerf.default.html) | [4605](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/SplitButtonMinimalPerf.default_base.html) | 0.99:1
CustomToolbarPrototype.default | [2823](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/CustomToolbarPrototype.default.html) | [2854](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/CustomToolbarPrototype.default_base.html) | 0.99:1
AlertMinimalPerf.default | [282](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/AlertMinimalPerf.default.html) | [288](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/AlertMinimalPerf.default_base.html) | 0.98:1
CheckboxMinimalPerf.default | [2783](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/CheckboxMinimalPerf.default.html) | [2835](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/CheckboxMinimalPerf.default_base.html) | 0.98:1
CardMinimalPerf.default | [550](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/CardMinimalPerf.default.html) | [568](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/CardMinimalPerf.default_base.html) | 0.97:1
ChatDuplicateMessagesPerf.default | [295](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ChatDuplicateMessagesPerf.default.html) | [307](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ChatDuplicateMessagesPerf.default_base.html) | 0.96:1
ImageMinimalPerf.default | [363](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ImageMinimalPerf.default.html) | [379](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ImageMinimalPerf.default_base.html) | 0.96:1
ListNestedPerf.default | [550](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ListNestedPerf.default.html) | [571](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ListNestedPerf.default_base.html) | 0.96:1
RadioGroupMinimalPerf.default | [423](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/RadioGroupMinimalPerf.default.html) | [442](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/RadioGroupMinimalPerf.default_base.html) | 0.96:1
SkeletonMinimalPerf.default | [346](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/SkeletonMinimalPerf.default.html) | [360](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/SkeletonMinimalPerf.default_base.html) | 0.96:1
TableMinimalPerf.default | [417](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/TableMinimalPerf.default.html) | [435](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/TableMinimalPerf.default_base.html) | 0.96:1
AccordionMinimalPerf.default | [148](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/AccordionMinimalPerf.default.html) | [155](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/AccordionMinimalPerf.default_base.html) | 0.95:1
LabelMinimalPerf.default | [387](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/LabelMinimalPerf.default.html) | [406](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/LabelMinimalPerf.default_base.html) | 0.95:1
ListCommonPerf.default | [664](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ListCommonPerf.default.html) | [698](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ListCommonPerf.default_base.html) | 0.95:1
TextAreaMinimalPerf.default | [490](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/TextAreaMinimalPerf.default.html) | [517](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/TextAreaMinimalPerf.default_base.html) | 0.95:1
BoxMinimalPerf.default | [331](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/BoxMinimalPerf.default.html) | [351](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/BoxMinimalPerf.default_base.html) | 0.94:1
DialogMinimalPerf.default | [782](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/DialogMinimalPerf.default.html) | [834](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/DialogMinimalPerf.default_base.html) | 0.94:1
ProviderMinimalPerf.default | [395](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ProviderMinimalPerf.default.html) | [419](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/ProviderMinimalPerf.default_base.html) | 0.94:1
RefMinimalPerf.default | [241](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/RefMinimalPerf.default.html) | [257](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/RefMinimalPerf.default_base.html) | 0.94:1
TextMinimalPerf.default | [334](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/TextMinimalPerf.default.html) | [372](https://fluentuipr.z22.web.core.windows.net/pull/23021/perf-test-northstar/TextMinimalPerf.default_base.html) | 0.9:1

</details><div id="perfComment9424"/>

**Review by ling1726 — APPROVED** [review]

File: packages/fluentui/CHANGELOG.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Fix `Dialog` to keep it open when press click on content and release outside @yuanboxue-amber ([#22849](https://github.com/microsoft/fluentui/pull/22849))
 - Align Avatar `font-weight` between v0 and v9 @annabratseiko ([#22822](https://github.com/microsoft/fluentui/pull/22822))
 - Fix `Pill` to be selectable by keyboard @chpalac ([#22839](https://github.com/microsoft/fluentui/pull/22839))
+- Fixing the `MicPulseOff` outline path to be what's in the Figma file, it is currently the value of the filled icon @sfilipi([23021](https://github.com/microsoft/fluentui/pull/23021))
```
Comment (ling1726) [inline packages/fluentui/CHANGELOG.md:42]:
```suggestion
- Fixing the `MicPulseOff` outline path to be what's in the Figma file, it is currently the value of the filled icon @sfilipi ([23021](https://github.com/microsoft/fluentui/pull/23021))
```

File: packages/fluentui/CHANGELOG.md [hunk 2]
```diff
@@ -39,6 +39,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Fix `Dialog` to keep it open when press click on content and release outside @yuanboxue-amber ([#22849](https://github.com/microsoft/fluentui/pull/22849))
 - Align Avatar `font-weight` between v0 and v9 @annabratseiko ([#22822](https://github.com/microsoft/fluentui/pull/22822))
 - Fix `Pill` to be selectable by keyboard @chpalac ([#22839](https://github.com/microsoft/fluentui/pull/22839))
+- Fix `MicPulseOff` outline path to align with Figma @sfilipi ([23021](https://github.com/microsoft/fluentui/pull/23021))
```
Comment (ling1726) [inline packages/fluentui/CHANGELOG.md:42]:
```suggestion
	- Fixing the `MicPulseOff` outline path to be what's in the Figma file, it is currently the value of the filled icon @sfilipi ([#23021](https://github.com/microsoft/fluentui/pull/23021)
```

**Pull request merged**